### PR TITLE
{lang}[GCCcore/11.2.0] Python v2.7.18

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['da3080e3b488f648a3d7a4560ddee895284c3380b11d6de75edb986526b9a814']
+
+dependencies = [
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:15]: Probably missing buliddep on pkg-config here, at least the Py3 for 11.2.0 has one

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+}
+
+# order is important!
+# package versions updated 8 January 2022, to the most recent versions still supporting Python 2.x
+exts_list = [
+    ('setuptools', '44.1.1', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'checksums': ['c67aa55db532a0dadc4d2e20ba9961cbd3ccc84d544e9029699822542b5a476b'],
+    }),
+    ('wheel', '0.37.1', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:43]: I would use the same wheel as we have for the Py3 with 11.2.0, i.e. 0.36.2

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+    ('Cython', '0.29.26', {
+        'checksums': ['af377d543a762867da11fcf6e558f7a4a535ff8693f30cce123fab10c00fa312'],
+    }),
+    ('six', '1.16.0', {
+        'checksums': ['1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926'],
+    }),
+    ('toml', '0.10.2', {
+        'checksums': ['b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f'],
+    }),
+    ('setuptools_scm', '4.1.2', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:70]: According to pypi the latest to support 2.7 is 5.0.2

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,422 @@
...
+    ('ipaddress', '1.0.23', {
+        'checksums': ['b7f8e0369580bb4a24d5ba1d7cc29660a4a6987763faf1d8a8046830e020e7e2'],
+    }),
+    ('asn1crypto', '1.4.0', {
+        'checksums': ['f4f6e119474e58e04a2b1af817eb585b4fd72bdd89b998624712b5c99be7641c'],
+    }),
+    ('idna', '2.10', {
+        'checksums': ['b307872f855b18632ce0c21c5e45be78c0ea7ae4c15c828c20788b26921eb3f6'],
+    }),
+    ('pycparser', '2.21', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:112]: Py3 for 11.2.0 uses 2.20

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 5]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['e644fdec12f7872f86c58ff790da456218b10f863970249516d60a5eaca77206'],
+    }),
+    ('cffi', '1.15.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:115]: py3 for 11.2.0 uses 1.14.6

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 6]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['aef77c9fb94a3ac588e87841208bdec464471d9871bd5050a287cc9a475cd0ba'],
+    }),
+    ('PyNaCl', '1.4.0', {
+        'modulename': 'nacl',
+        'checksums': ['54e9a2c849c742006516ad56a88f5c74bf2ce92c9f67435187c3c5953b346505'],
+    }),
+    ('bcrypt', '3.1.7', {
+        'checksums': ['0b0069c752ec14172c5f78208f1863d7ad6755a6fae6fe76ec2c80d13be41e42'],
+    }),
+    ('paramiko', '2.9.1', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:131]: py3 for 11.2.0 uses 2.7.2

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 7]
```diff
@@ -0,0 +1,422 @@
...
+    ('pytz', '2021.3', {
+        'checksums': ['acad2d8b20a1af07d4e4c9d2e9285c5ed9104354062f275f3fcd88dcef4f1326'],
+    }),
+    ('bitstring', '3.1.9', {
+        'checksums': ['a5848a3f63111785224dca8bb4c0a75b62ecdef56a042c8d6be74b16f7e860e7'],
+    }),
+    ('appdirs', '1.4.4', {
+        'checksums': ['7d5d0167b2b1ba821647616af46a749d1c653740dd0d2415100fe26e27afdf41'],
+    }),
+    ('distlib', '0.3.4', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:158]: py3 for 11.2.0 uses 0.3.2

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 8]
```diff
@@ -0,0 +1,422 @@
...
+        'source_tmpl': 'distlib-%(version)s.zip',
+        'checksums': ['e4b58818180336dc9c529bfb9a0b58728ffc09ad92027a3f30b7cd91e3458579'],
+    }),
+    ('filelock', '3.2.1', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:162]: py3 for 11.2.0 uses 3.0.12

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 9]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['9cdd29c411ab196cf4c35a1da684f7b9da723696cb356efa45bf5eb1ff313ee3'],
+    }),
+    ('importlib_resources', '3.3.1', {
+        'checksums': ['0ed250dbd291947d1a298e89f39afcc477d5a6624770503034b72588601bcc05'],
+    }),
+    ('platformdirs', '2.0.2', {
+        'checksums': ['3b00d081227d9037bbbca521a5787796b5ef5000faea1e43fd76f1d44b06fcfa'],
+    }),
+    ('virtualenv', '20.13.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:171]: py3 for 11.2.0 uses 20.7.0

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 10]
```diff
@@ -0,0 +1,422 @@
...
+    ('docopt', '0.6.2', {
+        'checksums': ['49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491'],
+    }),
+    ('joblib', '0.14.1', {
+        'checksums': ['0630eea4f5664c463f23fbf5dcfc54a2bc6168902719fa8e19daf033022786c8'],
+    }),
+    ('chardet', '4.0.0', {
+        'checksums': ['0d6f53a15db4120f2b08c94f11e7d93d2c911ee118b6b30a04ec3ee8310179fa'],
+    }),
+    ('certifi', '2021.10.8', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:183]: py3 for 11.2.0 uses 2021.5.30

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 11]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['78884e7c1d4b00ce3cea67b44566851c4343c120abd683433ce934a68ea58872'],
+    }),
+    ('urllib3', '1.26.8', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:186]: py3 for 11.2.0 uses 1.26.6

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 12]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['0e7c33d9a63e7ddfcb86780aac87befc2fbddf46c58dbb487e0855f7ceec283c'],
+    }),
+    ('requests', '2.27.1', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:189]: py3 for 11.2.0 uses 2.26.0

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 13]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['68d7c56fd5a8999887728ef304a6d12edc7be74f1cfa47714fc8b414525c9a61'],
+    }),
+    ('xlrd', '2.0.1', {
+        'checksums': ['f72f148f54442c6b056bf931dbc34f986fd0c3b0b6b5a58d013c9aef274d0c88'],
+    }),
+    ('py_expression_eval', '0.3.14', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:195]: py3 for 11.2.0 uses 0.3.13

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 14]
```diff
@@ -0,0 +1,422 @@
...
+    ('tabulate', '0.8.9', {
+        'checksums': ['eb1d13f25760052e8931f2ef80aaf6045a6cceb47514db8beab24cded16f13a7'],
+    }),
+    ('ujson', '2.0.3', {
+        'checksums': ['bd2deffc983827510e5145fb66e4cc0f577480c62fe0b4882139f8f7d27ae9a3'],
+    }),
+    ('atomicwrites', '1.4.0', {
+        'checksums': ['ae70396ad1a434f9c7046fd2dd196fc04b12f9e91ffb859164193be8b6168a7a'],
+    }),
+    ('py', '1.11.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:207]: py3 for 11.2.0 uses 1.10.0

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 15]
```diff
@@ -0,0 +1,422 @@
...
+    ('zipp', '1.2.0', {
+        'checksums': ['c70410551488251b0fee67b460fb9a536af8d6f9f008ad10ac51f615b6a521b1'],
+    }),
+    ('configparser', '4.0.2', {
+        'checksums': ['c7d282687a5308319bf3d2e7706e575c635b0a470342641c93bea0ea3b5331df'],
+    }),
+    ('contextlib2', '0.6.0.post1', {
+        'checksums': ['01f490098c18b19d2bd5bb5dc445b2054d2fa97f09a4280ba2c5f3c394c8162e'],
+    }),
+    ('importlib_metadata', '1.7.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:225]: according to pypi 2.1.2 is the latest for py2.7

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 16]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['90bb658cdbbf6d1735b6341ce708fc7024a3e14e99ffdc5783edea9f9b077f83'],
+    }),
+    ('pluggy', '0.13.1', {
+        'checksums': ['15b2acde666561e1298d71b523007ed7364de07029219b604cf808bfa1c765b0'],
+    }),
+    ('more-itertools', '5.0.0', {
+        'checksums': ['38a936c0a6d98a38bcc2d03fdaaedaba9f412879461dd2ceff8d37564d6522e4'],
+    }),
+    ('attrs', '21.4.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:234]: py3 for 11.2.0 uses 21.2.0

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 17]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['5b327ac1320dc863dca72f4514ecc086f31186744b84a230374cc1fd776feae5'],
+    }),
+    ('sphinxcontrib-websupport', '1.1.2', {
+        'modulename': 'sphinxcontrib.websupport',
+        'checksums': ['1501befb0fdf1d1c29a800fdbf4ef5dc5369377300ddbdd16d2cd40e54c6eefc'],
+    }),
+    ('Pygments', '2.5.2', {
+        'checksums': ['98c8aa5a9f778fcd1026a17361ddaf7330d1b7c62ae97c3bb0ae73e0b9b6b0fe'],
+    }),
+    ('imagesize', '1.3.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:264]: py3 for 11.2.0 uses 1.2.0

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 18]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['cd1750d452385ca327479d45b64d9c7729ecf0b3969a58148298c77092261f9d'],
+    }),
+    ('docutils', '0.17.1', {
+        'checksums': ['686577d2e4c32380bb50cbb22f575ed742d58168cee37e99117a854bcd88f125'],
+    }),
+    ('snowballstemmer', '2.2.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:270]: pyu3 for 11.2.0 uses 2.1.0

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 19]
```diff
@@ -0,0 +1,422 @@
...
+    ('sphinx-bootstrap-theme', '0.8.0', {
+        'checksums': ['038ee7e89478e064b5dd7e614de6f3f4cec81d9f9efbebb06e105693d6a50924'],
+    }),
+    ('colorama', '0.4.4', {
+        'checksums': ['5941b2b48a20143d2267e95b1c2a7603ce057ee39fd88e7329b0c292aa16869b'],
+    }),
+    ('click', '7.1.2', {
+        'checksums': ['d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a'],
+    }),
+    ('psutil', '5.9.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:294]: py3 for 11.2.0 uses 5.8.0

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 20]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['902b1b88936918f9b2a19e0e5eb7ccb430ae45cde4f39ea4b36932920d33952d'],
+    }),
+    ('functools32', '3.2.3-2', {
+        'checksums': ['f6253dfbe0538ad2e387bd8fdfd9293c925d63553f5813c4e587745416501e6d'],
+    }),
+    ('futures', '3.3.0', {
+        'modulename': False,
+        'checksums': ['7e033af76a5e35f58e56da7a91e687706faf4e7bdfb2cbc3f2cca6b9bcda9794'],
+    }),
+    ('glob2', '0.6', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:316]: are you sure 0.7 won't work?

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 21]
```diff
@@ -0,0 +1,422 @@
...
+    }),
+    ('keyring', '18.0.1', {
+        'modulename': False,  # Doesn't work properly if HOME directory contains keys
+        'checksums': ['67d6cc0132bd77922725fae9f18366bb314fd8f95ff4d323a4df41890a96a838'],
+    }),
+    ('keyrings.alt', '3.2.0', {
+        'modulename': False,  # Doesn't work properly if HOME directory contains keys
+        'checksums': ['1c9981c351dabe902172ccf75bccff78185548f15ad51d5297e6366c0f4c3b51'],
+    }),
+    ('tomlkit', '0.7.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:358]: according to pypi 0.7.2 is the latest for py2.7 which is what py3 for 11.2.0 uses

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 22]
```diff
@@ -0,0 +1,422 @@
...
+    ('shellingham', '1.4.0', {
+        'checksums': ['4855c2458d6904829bd34c299f11fdeed7cfefbf8a2c522e4caea6cd76b3171e'],
+    }),
+    ('requests-toolbelt', '0.9.1', {
+        'checksums': ['968089d4584ad4ad7c171454f0a5c6dac23971e9472521ea3b6d49d610aa6fc0'],
+    }),
+    ('pyrsistent', '0.16.1', {
+        'checksums': ['aa2ae1c2e496f4d6777f869ea5de7166a8ccb9c2e06ebcf6c7ff1b670c98c5ef'],
+    }),
+    ('pkginfo', '1.8.2', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:371]: py3 for 11.2.0 uses 1.7.1

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 23]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['542e0d0b6750e2e21c20179803e40ab50598d8066d51097a0e382cba9eb02bff'],
+    }),
+    ('pexpect', '4.8.0', {
+        'checksums': ['fc65a43959d153d0114afe13997d439c22823a27cefceb5ff35c2178c6784c0c'],
+    }),
+    ('jsonschema', '3.2.0', {
+        'checksums': ['c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a'],
+    }),
+    ('simplejson', '3.17.6', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:380]: py3 for 11.2.0 uses 3.17.3

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 24]
```diff
@@ -0,0 +1,422 @@
...
+        'checksums': ['be9aa45477a134aee56c8fac518627e1154df063e85f67d4f83ce0ccc23688e8'],
+    }),
+    ('ptyprocess', '0.7.0', {
+        'source_tmpl': SOURCE_WHL,
+        'checksums': ['4b41f3967fce3af57cc7e94b888626c18bf37a083e3651ca8feeb66d492fef35'],
+    }),
+    ('lockfile', '0.12.2', {
+        'checksums': ['6aed02de03cba24efabcd600b30540140634fc06cfa603822d508d5361e9f799'],
+    }),
+    ('poetry-core', '1.0.7', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:409]: py3 for 11.2.0 uses 1.0.3

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 25]
```diff
@@ -0,0 +1,422 @@
...
+        'modulename': 'poetry.core',
+        'checksums': ['98c11c755a16ef6c5673c22ca94a3802a7df4746a0853a70b6fae8b9f5cac206'],
+    }),
+    ('poetry', '1.1.12', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:413]: py3 for 11.2.0 uses 1.1.7

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb (hunk 3 above)
Comment (boegel) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:70]: Yes, but an older `setuptools_scm` is required for one of the extensions below.

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb (hunk 15 above)
Comment (boegel) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:225]: Here too, an older `importlib_metadata` is required for one of the extensions below

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb (hunk 20 above)
Comment (boegel) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:316]: Yes, won't work, glob <= 0.6 is required by one of the extensions below

**Comment by boegel** [comment]
@akesandgren I'm not convinced that we should downgrade the extesions to be in line with the Python 3 easyconfig (where possible), do you think that's worth the trouble?

**Comment by akesandgren** [comment]
I have a .eb with the above changes under test right now.
I do think we should stay in sync for extensions between py3 and py2 under a specific toolchain version as much as possible. It makes things less prone to give difficult differences in behaviour when people try both.
And py2 should go away any ~~year~~ ~~decade~~ century now...

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb (hunk 15 above)
Comment (akesandgren) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:225]: ignore this one, poetry-core requires importlib-metadata <2

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb (hunk 3 above)
Comment (akesandgren) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:70]: Not according to the versions I have in my .eb I use 5.0.2 and have no problems

Comment (boegel) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb]: Hmm, ok then. I could've sworn there was a reason I stuck to an older version here...

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 26]
```diff
@@ -0,0 +1,430 @@
...
+        'checksums': ['b307872f855b18632ce0c21c5e45be78c0ea7ae4c15c828c20788b26921eb3f6'],
+    }),
+    ('pycparser', '2.20', {
+        'checksums': ['2d475327684562c3a96cc71adf7dc8c4f0565175cf86b6d7a404ff4c771f15f0'],
+    }),
+    ('cffi', '1.14.6', {
+        'checksums': ['c9a875ce9d7fe32887784274dd533c57909b7b1dcadcc128a2ac21331a9765dd'],
+    }),
+    ('cryptography', '3.3.2', {
+        'preinstallopts': "export CARGO_HOME=%(builddir)s/cargo && ",
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:124]: This line shouldn't be here, that only makes sense for cryptography versions 3.4.x and newer (which require Rust as a build dependency)

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb (hunk 26 above)
Comment (boegel) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb]: removed in f55296948b

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2624.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/a79806d820cde527614120e3884124c2 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/6fbc1e61d8c0e8f6e7522a1f3c0e4559 for a full test report.

**Comment by boegel** [comment]
> I do think we should stay in sync for extensions between py3 and py2 under a specific toolchain version as much as possible. It makes things less prone to give difficult differences in behaviour when people try both. And py2 should go away any ~year~ ~decade~ century now...

Yes, good point, makes sense. Thanks for making those changes already in 10bfb55

This PR is good to go now as far as I can tell...

File: easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb [hunk 27]
```diff
@@ -0,0 +1,429 @@
...
+    ('filelock', '3.0.12', {
+        'checksums': ['18d82244ee114f543149c66a6e0c14e9c4f8a1044b5cdaadd0f82159d6a6ff59'],
+    }),
+    ('importlib_resources', '3.3.1', {
+        'checksums': ['0ed250dbd291947d1a298e89f39afcc477d5a6624770503034b72588601bcc05'],
+    }),
+    ('platformdirs', '2.0.2', {
+        'checksums': ['3b00d081227d9037bbbca521a5787796b5ef5000faea1e43fd76f1d44b06fcfa'],
+    }),
+    ('backports.entry_points_selectable', '1.1.1', {
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:176]: Oh bummer I forgot to double check that one...

[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb:176]: @boegel See https://github.com/boegel/easybuild-easyconfigs/pull/86

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3502.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/206bb5cc231fab1e8def9ba4e7cdf353 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**